data_managers/data_manager_motus_db_downloader/data_manager/motus_db_fetcher.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<tool id="motus_db_fetcher" name="mOTUs DB fetcher" tool_type="manage_data" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
+    <description>Fetches the DB required for mOTUs</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="biotools" />
+    <expand macro="requirements" />
+    <!-- <expand macro="version" /> -->
+
+    <command detect_errors="exit_code">
+    <![CDATA[
+        python '$__tool_directory__/data_manager_fetch_motus_db.py' --out '${out_file}' --version '${version}' 
+        $test_data_manager
+    ]]>
+    </command>
+    <inputs>
+    <param name="test_data_manager" type="boolean" truevalue="--test" falsevalue="" checked="False" label="Copy minimal test DB and create mock data table entry." />
+
+    <!-- <param name="test_data_manager" type="text" value=""/> -->
+    <param argument="--version" type="select" multiple="false" label="Database Version" help="Check https://zenodo.org/records/7778108 for current version. The default value is recommended.">
```

Comment:
You couls use a text param here with options.

```
<param name="foo" type="text" value="foo 1">
    <option value="foo 1">Foo 1 Display</option>
    <option value="foo 2">Foo 2 Display</option>
</param>

Reply:
But this will allow users to add different versions (that cannot be downloaded since the links do not work). So I would prefer to limit the choice using the type="select"

Reply:
True, but DMs are not for users, they are for admins - we assume they now what they are doing.

The downside of not allowing free-text inputs is that for every new release we need to update the DM - which is a lot of work.

Reply:
While I do agree, we also need to make sure that data tables are populated with the same conventions between servers, to make workflows that depend on those DBs federated usable. I also think for tools that can work with many DBs one needs to be more flexible, but for tools that usually have only one DB per version associated, a more strict enforcement of data table entries is needed.